### DISCUSSION TITLE
Video games

### DISCUSSION THREAD

**Discussion by ellielo** [NAIST-SE/AttractingContributors#369] [General]
Please share what you are currently playing and what is your favorite of all the times.

**Comment by janply** [comment]
Counter Strike for ever.

**Comment by hernessteve** [comment]
I play a lot and definitely have a lot of favorites. At the moment i enjoy playing Friday Night Funkin Games on [this site](https://ojigames.com/friday-night-funkin-games/). Doki Doki Takeover, FNF vs Minecraft Steve, Baldi Vs Huggy Wuggy Sings Expurgation - those are only few of them but the list is huge and will satisfy many different tastes.

**Comment by McJul** [comment]
I'm playing wordle lol
